package.json

```diff
@@ -46,6 +46,7 @@
     "watch": "yarn build --watch"
   },
   "devDependencies": {
+    "@superfaceai/parser": "^1.2.0",
```

Comment:
question: is `^` intentional? I know it is not production dependency now, so should be fine. But asking to be sure.

Reply:
I think it is intentional- we use parser in tests

Reply:
I know to create profileId using ProfileId and ProfileVersion. But I was pointing added `^`, but dev dependencies will use yarn.lock so it is fine.